tests/lib/nested.sh

```diff
@@ -1111,7 +1111,10 @@ nested_start_classic_vm() {
 }
 
 nested_destroy_vm() {
-    systemd_stop_and_destroy_unit "$NESTED_VM"
+    # XXX: is this the right way to do this ???
+    systemctl stop "$NESTED_VM"
```

Comment:
There is now "systemd_stop_and_remove_unit", see #9601 and #9606

Reply:
thanks